app/views/key_pairs/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% title _("Key pairs for: %s") % @compute_resource.name %>
+
+<table class="<%= table_css_classes 'table-fixed' %>">
+  <% if @key_pairs.any? %>
+  <thead>
+  <tr>
+    <th width="6%"><%= _('Active key') %></th>
+    <th width="40%"><%= _('Name') %></th>
```

Comment:
When looking closely, the "Active" icon seems a little bigger from the other icons.
When it was on a separate column, I did not notice that, now I do.